xapian-bindings/android/aar/.gitignore

```diff
@@ -0,0 +1,3 @@
+/jni/
+/res/
+classes.jar
```

Comment:
When you see a little symbol like this in github, it means the file doesn't have an ending newline. It's generally better to have one, where file formats allow it (which .gitignore, XML &c do).